### DISCUSSION TITLE
allowing just anybody to "Resolve" conversations

### DISCUSSION THREAD

**Discussion by TeoTwawki** [community/community#18458] [Pull Requests]
This is extremely annoying when you are commenting or requesting changes and they keep reclosing it and co devs miss the messages because of it. I would rather only project members be able to do this. 

if nothing else this should a setting like we have for github wiki's.

It's even worse than I originally thought: outsiders (not even the PR author!) can start a conversation, others get involved, and then the non member that started it can close it before anyone has acted on the changes discussed. If i have to expand every single closed conversation just in case, the feature wouldn't be serving its purpose.

**Comment by efjacobson** [comment]
+1

i would love to see a setting that would make it so that only the person who started a conversation is able to resolve it

**Reply by TeoTwawki** [reply]
I'm looking to restrict it to project members, but even that would be a massive improvement.

**Comment by TeoTwawki** [comment]
This is still a problem.
